e2e/flows/set-default-owner-and-scope.e2e.4.ts

```diff
@@ -0,0 +1,116 @@
+describe('set default owner and scope', function() {
+  this.timeout(0);
+  let helper: Helper;
+  before(() => {
+    const helperOptions: HelperOptions = {
+      scopesOptions: {
+        remoteScopeWithDot: true
+      }
+    };
+    helper = new Helper(helperOptions);
+  });
+  after(() => {
+    helper.scopeHelper.destroy();
+  });
+  describe('author components', () => {
+    let parsedLinkOutput;
+    let defaultOwner;
+    let defaultScope;
+    let fullScope;
+    let componentId;
+    let componentPackageName;
+    before(() => {
+      helper.scopeHelper.initWorkspaceAndRemoteScope();
+      const remoteScopeParts = helper.scopes.remote.split('.');
+      defaultOwner = remoteScopeParts[0];
+      defaultScope = remoteScopeParts[1];
+      fullScope = `${defaultOwner}.${defaultScope}`;
+      componentId = `${fullScope}/utils/is-type`;
+      componentPackageName = `@${defaultOwner}/${defaultScope}.utils.is-type`;
+      helper.bitJsonc.addDefaultOwner(defaultOwner);
+      helper.bitJsonc.addDefaultScope(defaultScope);
+      helper.fixtures.populateWorkspaceWithUtilsIsType();
+      const rawLinkOutput = helper.command.link('-j');
+      parsedLinkOutput = JSON.parse(rawLinkOutput);
+      helper.command.tagAllComponents();
+    });
+    it('should create link with default owner as prefix', () => {
+      const linkFolderPath = path.normalize(`node_modules/${componentPackageName}`);
+      const linkFullPath = path.join(linkFolderPath, '/utils/is-type.js');
+      const outputLinkPath = parsedLinkOutput.linksResults[0].bound[0].to;
+      expect(outputLinkPath).to.equal(linkFullPath);
+      expect(path.join(helper.scopes.localPath, 'node_modules')).to.be.a.directory();
+      expect(path.join(helper.scopes.localPath, linkFolderPath)).to.be.a.directory();
+    });
+    describe('after export', () => {
+      let exportOutput;
+      before(() => {
+        exportOutput = helper.command.exportAllComponents();
+      });
+      it('should export the component to correct scope', () => {
+        expect(exportOutput).to.have.string('exported 1 components');
+        expect(exportOutput).to.have.string(`${defaultOwner}.${defaultScope}`);
+      });
+      describe('validate models data', () => {
+        let compModel;
+        let versionModel;
+        before(() => {
+          compModel = helper.command.catComponent(`utils/is-type`);
+          versionModel = helper.command.catComponent(`utils/is-type@latest`);
+        });
+        it('should store scope name with the format owner.scope in the models', () => {
+          expect(compModel.scope).to.equal(`${fullScope}`);
+        });
+        it('should store scope name with the format owner.scope in the models', () => {
+          expect(compModel.bindingPrefix).to.equal(`@${defaultOwner}`);
+          expect(versionModel.bindingPrefix).to.equal(`@${defaultOwner}`);
+        });
+      });
+      describe('on imported', () => {
```

Comment:
implies there is an on import event you are testing. Consider "post import"

Reply:
fixed